src/Admin.js

```diff
@@ -2,43 +2,70 @@ import Hotel from './Hotel'
 
 class Admin extends Hotel {
   constructor(customerData, roomsData, bookingsData, currentDate) {
-    super(customerData, roomsData, bookingsData)
+    super(customerData, roomsData, bookingsData, currentDate)
```

Comment:
I like this addition of current date as a property on the Hotel.  I have find date as a method, but this would pass down even easier!

Reply:
I agree.